src/main/java/cloud/orbit/spring/OrbitActorsProperties.java

```diff
@@ -28,19 +28,33 @@
 
 package cloud.orbit.spring;
 
-import cloud.orbit.actors.Stage;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+import cloud.orbit.actors.Stage;
+import cloud.orbit.actors.extensions.ActorExtension;
+
 import java.util.List;
 
 @ConfigurationProperties(prefix = "orbit.actors")
 public class OrbitActorsProperties {
+    private String basePackages;
     private String clusterName;
     private String nodeName;
     private Stage.StageMode stageMode;
+    private List<ActorExtension> extensions;
```

Comment:
I don't think this is the most appropriate way to pass what extensions we should be passing to the Stage. This class is meant to be instantiated from the Spring configuration runtime against the properties file. The Spring framework wouldn't know how to instantiate those ActorExtension's from the property file. In general these type of things would be obtained from the application context by looking up beans that have been instantiated using the ActorExtension interface and then to provide those to the Stage. This allows for the developer to have control over the instantiation of the extension and the autowiring that the framework has.